src/index.tsx

```diff
@@ -10,6 +10,8 @@ export {default as Touchable} from './touchable';
 export type {TouchableElement} from './touchable';
 export {default as Spinner} from './spinner';
 export {default as FadeIn} from './fade-in';
+export {default as DataCardAdvanced} from './data-card-advanced';
```

Comment:
would that include the test-screenshot?

Reply:
yes

Reply:
unfortunately trying to move the test-screenshot to the community folder did not work, I think I would have to create a new folder for it or some configuration, because of that I left it like this

Reply:
as I told you in the Blocks PR, you need to create the folder there, yes